.gitignore

```diff
@@ -1,2 +1,3 @@
 v1/lib/external/vendor
 .idea
+png
```

Comment:
git made this automatically when i clicked on ignore this dir.
I removed this because it is needed in the script, i just want to ignore the *.png files.